src/MenuStyle.php

```diff
@@ -94,6 +94,41 @@ class MenuStyle
      */
     private $coloursResetCode = "\033[0m";
 
+    /**
+     * @var int
+     */
+    private $borderTopWidth;
+
+    /**
+     * @var int
+     */
+    private $borderRightWidth;
+
+    /**
+     * @var int
+     */
+    private $borderBottomWidth;
+
+    /**
+     * @var int
+     */
+    private $borderLeftWidth;
+
+    /**
+     * @var string
+     */
+    private $borderColour = 'white';
```

Comment:
Can probably drop this assignment.

Reply:
If I drop it I get an error in generateBorderRows when the border colour isn't set.
Juste like the issue you had with bg and fg colours earlier.

Reply:
Hmm - It works fine for me as `private $borderColour`. It is set in the constructor from `$defaultStyleValues`